mevislab.github.io/content/tutorials/visualization/pathtracer/pathtracerexample2.md

```diff
@@ -0,0 +1,114 @@
+
+If you check your `SoExaminerViewer` you will see a black box, this indicates often a need for additional adjustments to color, light and transparency.
+
+![SoExaminerViewer](/images/tutorials/visualization/pathtracer/V6.2_3.png "SoExaminerViewer")
+
+To brighten up your scene, connect both the `SoPathTracerAreaLight` and `SoPathTracerBackgroundLight` modules to your `SoExaminerViewer` to improve scene lighting. 
+
+The `SoPathTracerAreaLight` shapes and colors specific areas realistically, while `SoPathTracerBackgroundLight` illuminates the whole scene with your chosen three colors. These lights make your scene vibrant, providing a solid base for further adjustments with other modules.
+
+![Lights](/images/tutorials/visualization/pathtracer/V6.2_4.png "Lights")
+
+Connect `SoLUTEditor` to your `SoPathTracerVolume` to visualize the knee. However, it might initially appear in black and with shading.
+
+![SoLUTEditor](/images/tutorials/visualization/pathtracer/V6.2_5_SoLUTEditor.png "SoLUTEditor")
+
+
+To easily identify your minimum and maximum range, connect the `MinMaxScan` module to your `LocalImage`. Open it's panel and read your *Static Min Value* and *Static Max Value*. This step allows you to set the appropriate values with convenience.
+
+![MinMaxScan](/images/tutorials/visualization/pathtracer/V6_MinMaxScan.png "MinMaxScan")
+
+Then open the panel of the `SoLUTEditor` and select tab *Range*. Define *New Range Min* as *0* and *New Range Max* as *2047*. Once done, manually define your desired colors for your LUT in the *Editor* tab.
+
+{{<alert class="info" caption="Additional Info">}}
+You can replace `SoLUTEditor` with `LUTLoad` and load the attached xml files to store the same adjustments.
+{{</alert>}}
+
+![LUT](/images/tutorials/visualization/pathtracer/V6.2_LUTandSoExaminerViewer.png "LUT")
+
+Now, let's enhance your rendering further by using the `SoPathTracerMaterial` module. This module provides essential material properties for geometry and volumes within the `SoPathTracer` scene. Open it’s panel and navigate to the tab *Surface Brdf*. Change the *Diffuse* color for altering the visual appearance of surfaces. The *Diffuse* color determines how light interacts with the surface, influencing its overall color and brightness.
+
+![SoPathTracerMaterial](/images/tutorials/visualization/pathtracer/V6.2_8.png "SoPathTracerMaterial")
+
+## Visualize Bones
+
+Follow the same steps for this scenario. Load this [Bones mask](/examples/visualization/example6/edited_Bones.mlimage) and examine it using `View2D`.  
+
+![Bones](/images/tutorials/visualization/pathtracer/V6.2_9.png "Bones")
+
+Start by refining the transparency of the cartilage structures using the `SoLUTEditor` to improve visibility and achieve a clearer depiction of the bones. Open it’s panel in the *Editor* tab, choose your *Index* number *3* and reduce the *Opacity* to *0*. 
```

Comment:
What index number 3? We did not even add the SoLutEditor. Where does the LUT come from?

Reply:
What is the Range? The LUT?